tests/test_hgmozilla.py

```diff
@@ -180,5 +180,30 @@ def test_fileinfo_release_channel(self):
         self.assertIsNotNone(info['netwerk/protocol/http/nsHttpConnectionMgr.cpp'])
 
 
+class AnnotateTest(unittest.TestCase):
+
+    def test_annotate(self):
+        path = 'netwerk/protocol/http/nsHttpConnectionMgr.cpp'
+        info = hgmozilla.Annotate.get(path)
+
+        self.assertIsNotNone(info)
+        self.assertIsNotNone(info['netwerk/protocol/http/nsHttpConnectionMgr.cpp'])
```

Comment:
Can you assert a couple of things here? E.g. that `abspath` is == netwerk/protocol/http/nsHttpConnectionMgr.cpp and that there is a `annotate` key in the dictionary